tests/callbacks/test_lr_monitor.py

```diff
@@ -150,7 +184,22 @@ def test_lr_monitor_param_groups(tmpdir):
     """ Test that learning rates are extracted and logged for single lr scheduler. """
     tutils.reset_seed()
 
-    model = EvalModelTemplate()
+    class LessBoringModel(BoringModel):
+        def __init__(self):
+            super().__init__()
+            self.learning_rate = 0.1
+
+        def configure_optimizers__param_groups(self):
+            param_groups = [
+                {'params': list(self.parameters())[:2], 'lr': self.learning_rate * 0.1},
+                {'params': list(self.parameters())[2:], 'lr': self.learning_rate}
+            ]
+
+            optimizer = optim.Adam(param_groups)
+            lr_scheduler = optim.lr_scheduler.StepLR(optimizer, 1, gamma=0.1)
+            return [optimizer], [lr_scheduler]
+
+    model = LessBoringModel()
     model.configure_optimizers = model.configure_optimizers__param_groups
```

Comment:
why this and not just override the method directly?

Reply:
Ahhh yeah, you are right!